src/deploy/campaign/get-config.ts

```diff
@@ -219,7 +225,10 @@ const validatePrice = (config : ICurvePriceConfig) => {
   return (priceB <= priceA);
 };
 
-export const buildCrosschainConfig = () : TZNSCrossConfig => {
+export const buildCrosschainConfig = (env : TEnvironment) : {
+  crosschain : TZNSCrossConfig;
+  srcChainName : TSupportedChain;
+} => {
   const srcChainName = process.env.SRC_CHAIN_NAME as TSupportedChain;
```

Comment:
This won't actually cause an error if `proess.env.SRC_CHAIN_NAME` is not one of the one's included in `TSupportedChain`, the `as` in TypeScript is basically telling the transpiler "even if this isn't what I type it as, don't complain because I know what I'm doing"

It will be fine though because your `switch` below checks for exact values